src/index.ts

```diff
@@ -182,32 +187,32 @@ const cs3browser: JupyterFrontEndPlugin<void> = {
         const cs3Panel = new Cs3Panel('cs3 panel', 'cs3-panel', kernelIcon);
 
         stateDB.save('share', {share_type: 'filelist'});
+        stateDB.save('showHidden', false);
```

Comment:
What is this stateDB? does it persist in the browser localstorage? If so, doesn't this reset the config everytime the user loads the page?

Reply:
stateDB is a handy storage to share info between components and yes, after refresh the data is destroyed, but after refresh it's loaded again when jl pulls file list

Reply:
Should we make it persistent in localstorage?

Reply:
Actually after double check the data persisted in stateDB is not affected by refreshments, I'm pretty sure it uses localstorage underneath. This is the default way to store vars in jupyter, I don't think we need to switch to localstorage explicitly.

Reply:
perfect!